src/panels/experimental-ui/hui-notification-card.js

```diff
@@ -0,0 +1,74 @@
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import computeDomain from '../../common/entity/compute_domain.js';
+
+import '../../cards/ha-persistent_notification-card.js';
+import './hui-error-card.js';
+
+class HuiNotifcationCard extends PolymerElement {
+  static get properties() {
+    return {
+      hass: {
+        type: Object,
+        observer: '_hassChanged',
+      },
+      config: {
+        type: Object,
+        observer: '_configChanged',
+      }
+    };
+  }
+
+  getCardSize() {
+    return 2;
+  }
+
+  _configChanged(config) {
+    this._entityId = null;
+    if (this.childNodes.length) {
+      this.removeChild(this.childNodes[0]);
+    }
+
+    let error = null;
+    let cardConfig;
+    let tag;
+
+    const entityId = config && config.entity;
+    if (entityId) {
+      if (computeDomain(entityId) === 'persistent_notification') {
+        this._entityId = entityId;
+        tag = 'ha-persistent_notification-card';
```

Comment:
We shouldn't have this one. Notifications come and go. We need a new system for it. Like a notification pane like Android